app/Schema/Mysql.php

```diff
@@ -5,7 +5,22 @@
 use PDO;
 use Kanboard\Core\Security;
 
-const VERSION = 93;
+const VERSION = 94;
+
+function version_94(PDO $pdo)
+{
+    $pdo->exec("
+        CREATE TABLE colors (
+            color_id VARCHAR(50) NOT NULL COMMENT 'Backwards compatible format with Kanboard color IDs.',
+            name VARCHAR(30) NOT NULL COMMENT 'Name shown. Can be changed to semantics: Important, Idea...',
+            background VARCHAR(25) NOT NULL COMMENT 'Any valid HTML way of representing color.',
+            border VARCHAR(25) NOT NULL COMMENT 'Any valid HTML way of representing color.',
+            is_usable TINYINT(1) NOT NULL DEFAULT '1',
```

Comment:
Just to follow the existing convention, can you rename `is_usable` to `is_active`?